test/performance/benchmarks/scale-from-zero/main.go

```diff
@@ -360,5 +360,12 @@ func checkSLA(results *vegeta.Metrics, p95min time.Duration, p95max time.Duratio
 		return fmt.Errorf("SLA 2 failed. Max latency is higher than %dms: %s", latencyMax, results.Latencies.Max)
 	}
 
+	// SLA 3: making sure the defined vegeta rates/total requests are met
```

Comment:
We should be more specific in the comment here, this is total requests not rates. Same for the previous comments.